File: LearnOpencv/1/python/main.py
import cv2 as cv

# 读取图片
image = cv.imread('cqss.png')

# 创建一个全屏窗口
cv.namedWindow('FullScreen', cv.WND_PROP_FULLSCREEN)
cv.setWindowProperty('FullScreen', cv.WND_PROP_FULLSCREEN, cv.WINDOW_FULLSCREEN)

# 显示图片
cv.imshow('FullScreen', image)

# 等待用户按键
cv.waitKey(0)

# 关闭窗口
cv.destroyAllWindows()

File: LearnOpencv/1/c++/main.cpp
#include <iostream>
#include <opencv2\highgui\highgui.hpp>
#include <opencv2\opencv.hpp>
using namespace std;
using namespace cv;


int main() {
    // 读取图像
    cv::Mat image = cv::imread("cqss.png"); 
    if (image.empty()) {
        std::cerr << "Error: Could not open or find the image!" << std::endl;
        return -1;
    }

    // 创建一个全屏窗口
    cv::namedWindow("FullScreen Image", cv::WINDOW_NORMAL);
    cv::setWindowProperty("FullScreen Image", cv::WND_PROP_FULLSCREEN, cv::WINDOW_FULLSCREEN);

    // 显示图像
    cv::imshow("FullScreen Image", image);

    // 等待按键
    cv::waitKey(0);

    // 关闭窗口
    cv::destroyAllWindows();
    return 0;
}
